pages/dkp/kommander/1.4/clusters/attach-cluster/index.md

```diff
@@ -82,32 +86,86 @@ users:
+
+You have these security options when attaching a cluster:
+
+- Attach a cluster with no additional security restrictions
+- Attach a cluster that has networking restrictions
+
+#### Attaching a cluster with no additional security
+
+Use this option when you want to attach a cluster that doesn't require additional access information.
+
+1. Select the **Attach Cluster** option to display the **Connection Information** dialog box.
+
+1. Paste a kubeconfig file into the field, or select the upload link below the field to specify a file.
 
-Selecting the **Attach Cluster** option displays the **Connection Information** dialog box. This dialog box accepts a kubeconfig file, that you can paste, or upload into the field. In the **Context** select list, you can select the intended context or change the display name provided with the config. You can add labels to classify your cluster and select the platform services to install.
+1. Select the intended context or change the display name provided with the config in the **Context** select list
 
-Platform services extend the functionality of Kubernetes and allow you to deploy ready-to-use logging and monitoring stacks by federating platform services when attaching a cluster to Kommander. For more information, refer to [workspace platform services][workspace_platform_services].
+1. Add labels to classify your cluster as needed.
+
+1. Select the platform services to install. Platform services extend the functionality of Kubernetes and allow you to deploy ready-to-use logging and monitoring stacks by federating platform services when attaching a cluster to Kommander. For more information, refer to [workspace platform services][workspace_platform_services].
+
+1. Select the **Submit** button to begin the cluster attachment processing.
 
 ![Add Cluster Connect](/dkp/kommander/1.4/img/add-cluster-connect.png)
 
+#### Attaching a cluster with networking restrictions
+
+Use this option when you want to attach a cluster that is behind a proxy server or a firewall, or that requires additional access information. This page gathers the information required to create a kubeconfig file for the kubetunnel between Kommander and the cluster you want to attach.
+
+1. Select the card, Attach Cluster with Networking Restrictions to display the configuration page.
+
+1. Enter the **Cluster Name** and select a **Workspace** from the dropdown list.
+
+1. Create one or more new Labels as needed.
+
+1. Enter the **Load Balancer Hostname** and its related **URL Path Prefix**. The **Hostname** field is optional.
+
+1. Select the **Root CA Certificate** from the list of available Secrets, and add and extra Annotations as needed.
+
+1. Select the **Save & Generate kubeconfig** button to generate the kubeconfig file for kubetunnel.
```

Comment:
Should we include the annotations field too ?

Reply:
It is there, I will fix the formatting so that it stands out.